web/pandas/community/blog/asv-pandas-grant.md

```diff
@@ -0,0 +1,133 @@
+[how to make an open-source contribution](https://tubedu.org/w/kjnHEg72j76StmSFmjzbnE),
+which we invite you to take a look at it.
+
+We learned a lot from the source code and gained immense knowledge about
+best practices and code quality through this grant. We have been working
+on: updating the code to improve the style to follow the PEP-8 guidelines,
+removing Python2 compatibility code and six dependencies, and finding unused
+code and removing it. We also learned about GitHub actions, and we started
+building the CI on GitHub actions for the asv package; for that we have been
+working on add linting with Flake8, testing with pytest, building docs, and
+running CI on different python versions.
+
+Additionally, we were able to identify bugs in the source code, review
+pull-request from other contributors, and create new issues, something we
+thought only maintainers could do but not contributors. Finally, not only
+is reviewing the code itself a learning experience, but also the structure
+and folder hierarchy in the project started to be more transparent.
+
+### Our experience
+For this grant, we had a fantastic Mentor, Marc Garcia. He was always
+willing to share his knowledge, explain unclear concepts and share helpful
+feedback. Whenever we would implement that feedback, it felt easier to work
+on more issues faster. We felt the growth from the time we started on this
+project, and we will carry it along as we contribute to more Open Source
+projects; this all goes back to Marc for his fantastic mentorship. It is
+also important to note that we received feedback from other contributors,
+stakeholders, and core devs during this process, which gave us a broader
+look at the work in Open Source projects.
+
+We also built a strong teamwork partnership. We helped each other a lot as
+we had numerous one-on-one calls to understand the tasks better. We always
+looked for ways to support each other from the technical side and encouraged
+each other when needed. For us, it was professional and human growth.
+
+### Running an Open Source Software Sprint
+The knowledge and experience acquired in this process allowed us to
+organize two virtual sprints. The events were carried out in the company
+of local PyLadies communities; the first one was on February 26th with
+[PyLadies Kampala](https://twitter.com/pyladieskla?s=11) and on March 21
+with [PyLadies Colombia](https://bit.ly/sprint-asv).
+
+While organizing these events, we learned how to organize and conduct a
+virtual Sprint. Some participants in the sprint ultimately had no idea
+about open source, and it was great explaining open source concepts and
+taking them through the Git workflow. Finally, they were able to make their
+first contribution. We learned how to follow up on contributors, helping
+them along the way until their PRs were merged and by reviewing their
+contributions on Github.
+
+The most outstanding achievement was mentoring new contributors and
```

Comment:
I think we discussed about having photos of the sprints in the blog post. Is this finally not happening?

Reply:
As we agreed by DM, we will not include photos in this post.